crates/loans/src/lib.rs

```diff
@@ -245,6 +241,8 @@ pub mod pallet {
         CodecError,
         /// Collateral is reserved and cannot be liquidated
         CollateralReserved,
+        /// Arithmetic underflow
+        ArithmeticUnderflow,
```

Comment:
let's not redefine this, instead let's return `ArithmeticError::Underflow` as we do in other pallets